package.json

```diff
@@ -21,18 +20,19 @@
   "scripts": {
     "lint": "tslint -p ./tsconfig.json",
     "start": "react-scripts start",
-    "clean": "rm -rf build",
-    "build": "GENERATE_SOURCEMAP=true react-scripts build",
+    "start:static": "npm run-script build && npm run-script prerender && npm run-script server",
+    "clean": "rm -rf ./build",
+    "build": "GENERATE_SOURCEMAP=${GENERATE_SOURCEMAP:-true} react-scripts build",
     "build:clean": "npm run-script clean && npm run-script build",
-    "prerender": "node ./script/entry prerender",
-    "server": "NODE_ENV=development node script/entry server/cli",
+    "prerender": "NODE_ENV=test node ./script/entry prerender",
```

Comment:
definitely don't want to hardcode this env, i'd default it to `production` if anything